### PR TITLE
pitch-black: init at unstable-2019-07-23

### PR DESCRIPTION
**Pull request by pasqui23**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/themes/pitch-black/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ lib, stdenvNoCC, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation rec{
+  pname = "Pitch-Black";
```
Comment (IvarWithoutBones) [inline pkgs/data/themes/pitch-black/default.nix:4]: A bit more consistent.
```suggestion
  pname = "pitch-black";
```
That's not a must though, i just think it looks a bit nicer.

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -23874,6 +23874,8 @@ with pkgs;
 
   penna = callPackage ../data/fonts/penna { };
 
+  Pitch-Black = callPackage ../data/themes/pitch-black {};
```
Comment (IvarWithoutBones) [inline pkgs/top-level/all-packages.nix:23877]: Same as my other comment

**Review by IvarWithoutBones — CHANGES_REQUESTED** [review]
Functionality wise this looks good :+1: 

Could you also run `nixpkgs-fmt` on this?

File: pkgs/data/themes/pitch-black/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  };
+
+  dontBuild = true;
+  installPhase = ''
+    rm LICENSE README.md
+    mkdir -p $out/share
+    mv GTK $out/share/themes
+    mv * $out/share
+  '';
+
```
Comment (SuperSandro2000) [inline pkgs/data/themes/pitch-black/default.nix:20]:
```suggestion
  version = "unstable-2019-07-23";

  src = fetchFromGitHub {
    repo = pname;
    owner = "freefreeno";
    rev = "d8039341419aef1157c030bf3d9237bd926e0b95";
    hash = "sha256-Rn3ZMBD6srIkYFNN3HT5JFP46Akodmeqz5tbV2/2ZDA=";
  };

  dontBuild = true;

  installPhase = ''
    rm LICENSE README.md
    mkdir -p $out/share
    mv GTK $out/share/themes
    mv * $out/share
  '';
```

File: pkgs/top-level/all-packages.nix (hunk 2 above)
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:23877]:
```suggestion
  pitch-black = callPackage ../data/themes/pitch-black {};
```

**Review by SuperSandro2000 — APPROVED** [review]

File: pkgs/data/themes/pitch-black/default.nix [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  installPhase = ''
+    rm LICENSE README.md
+    mkdir -p $out/share
+    mv GTK $out/share/themes
+    mv * $out/share
+  '';
+
+  meta = with lib; {
+    description = "Pitch-Black theme is a dark plasma theme built with usability in mind and it looks great.";
```
Comment (IvarWithoutBones) [inline pkgs/data/themes/pitch-black/default.nix:24]: There shouldn't be a dot at the end of this, see [contributing.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes).
```suggestion
    description = "Pitch-Black is a dark plasma theme built with usability in mind";
```
The last part also seems a bit subjective

**Pull request merged**